app/src/main/java/com/example/coinage/fragments/profile/ProfileFragment.java

```diff
@@ -39,13 +43,32 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
+        tvName = view.findViewById(R.id.tvName);
+        tvName.setText(ParseUser.getCurrentUser().getUsername());
+
         btnLogout = view.findViewById(R.id.btnLogout);
         btnLogout.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 logoutUser();
             }
         });
+
+        btnEditInfo = view.findViewById(R.id.btnEditInfo);
+        btnEditInfo.setOnClickListener(new View.OnClickListener() {
```

Comment:
Cool thing - you can be a lot more concise here using a cool java feature called functional interfaces https://jenkov.com/tutorials/java-functional-programming/functional-interfaces.html#:~:text=A%20functional%20interface%20in%20Java,to%20the%20single%20unimplemented%20method.

see if you can figure out what I mean (hint - you don't have to type View.OnClickListener)  but don't spend 2 much time

Reply:
I think I got it, something like this?

define this outside of onViewCreated:
public interface onClickListener{
        public default void onClick(View v) {}
    }

and use it like this:
btnEditInfo.setOnClickListener(onClickListener -> goEditInfo());

Reply:
btnEditInfo.setOnClickListener((View v) -> {
    goEditInfo()
});